src/nxdoc/web-ui/nuxeo-elements/nuxeo-elements-tutorial.md

```diff
@@ -511,131 +537,150 @@ The final app should look like this:
-          <p>Repository: [[document.repository]]</p>
-          <p>State: [[document.state]]</p>
-        </div>
-        <div class="flex">
-          <h3>Upload files:</h3>
-          <nuxeo-file id="file" value="\{{blob}}"></nuxeo-file>
-        </div>
-      </div>
-      <div class="horizontal layout">
-        <div class="flex">
-          <h3>Contributors:</h3>
-          <ul>
-            <template is="dom-repeat" items="[[document.properties.dc:contributors]]" as="contributor">
-              <li>[[contributor]]</li>
+{{#> panel type='code' heading='src/my-doc-reader.js'}}
+```js
+import { html, PolymerElement } from "@polymer/polymer/polymer-element.js";
+import './shared-styles.js';
+import '@nuxeo/nuxeo-elements/nuxeo-connection.js';
+import '@nuxeo/nuxeo-elements/nuxeo-document.js';
+import '@nuxeo/nuxeo-elements/nuxeo-page-provider.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-path-suggestion/nuxeo-path-suggestion.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-data-table/iron-data-table.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-file.js';
+
+/**
+ * `my-doc-reader`
+ *
+ *
+ * @customElement
+ * @polymer
+ * @demo demo/index.html
+ */
+class MyDocReader extends PolymerElement {
+  static get template() {
+    return html`
+      <style>
+        :host {
+          display: block;
+        }
+        nuxeo-data-table {
+          height: 220px;
+        }
+      </style>
+
+      <nuxeo-connection
+        id="nxcon"
+        url="http://localhost:8080/nuxeo"
+        username="Administrator"
+        password="Administrator"
+      ></nuxeo-connection>
+
+      <nuxeo-document
+        id="doc"
+        auto
+        doc-path="[[targetPath]]"
+        response="{{document}}"
+      ></nuxeo-document>
+
+      <nuxeo-page-provider
+        auto
+        provider="advanced_document_content"
+        params="[[_computeParams(document)]]"
+        current-page="{{children}}"
+        enrichers="thumbnail, permissions"
+      >
+      </nuxeo-page-provider>
+
+      <div class="card">
+        <div class="circle">4</div>
+        <h1>Doc Reader</h1>
+
+        <h2>Title: [[document.title]]</h2>
+        <p>ID: [[document.uid]]</p>
+        <p>Repository: [[document.repository]]</p>
+        <p>State: [[document.state]]</p>
+        <h3>Contributors:</h3>
+        <ul>
+          <template
+            is="dom-repeat"
+            items="[[document.properties.dc:contributors]]"
+            as="contributor"
+          >
+            <li>[[contributor]]</li>
+          </template>
+        </ul>
+        <h3>Children:</h3>
+        <ul>
+          <template is="dom-repeat" items="[[children]]" as="child">
+            <li>[[child.title]]</li>
+          </template>
+        </ul>
+        <nuxeo-path-suggestion label="document path" value="{{targetPath}}"></nuxeo-path-suggestion>
+        <h3>Children:</h3>
+        <nuxeo-data-table items=[[children]]>
+        <nuxeo-data-table-column name="Icon">
+            <template>
```

Comment:
Two different spacings here (2 and 4 spaces).
(there are more occurrences of this too)